CMakeLists.txt

```diff
@@ -1,127 +1,178 @@
+## ---------------------------------------------------------------------
 ##
-# Deal.II Swiss Army Knife
+## Copyright (C) 2014 - 2015 by the deal2lkit authors
 ##
+## This file is part of the deal2lkit library.
```

Comment:
Please acknowledge Matthias and the deal.II library, by saying that the structure of this cmake has been copied blah, blah...